src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php

```diff
@@ -157,6 +157,8 @@ public function configureModuleAction($module_name)
     }
 
     /**
+     * @AdminSecurity("is_granted('create', 'ADMINMODULESSF_') || is_granted('update', 'ADMINMODULESSF_') || is_granted('delete', 'ADMINMODULESSF_')")
```

Comment:
Are you sure about the "ADMINMODULESSF_" ?

Reply:
I noticed in some places that write the permission manual that they use the controller name instead like `AdminModulesSf` Wouldn't it work as well? Even all the values that we put in `_legacy_controller` don't look like this

Maybe both format can work, but the one matching the controller in camel case feels better no?